Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -435,7 +435,7 @@ extension LoginListViewController: LoginViewModelDelegate {
 
     func breachPathDidUpdate() {
         DispatchQueue.main.async {
-            self.tableView.reloadRows(at: self.viewModel.breachIndexPath, with: .right)
+            self.tableView.reloadRows(at: Array(self.viewModel.breachIndexPath), with: .none)
```

Comment:
A faster way to handle this, instead of looping over each item in the set to convert it to an array and then looping over that array again to reload rows, may be to use a `forEach` loop over the set:

```
self.viewModel.breachIndexPath.forEach {
    self.tableView.reloadRows(at: [$0], with: .none)
}
```